mmv1/third_party/terraform/resources/resource_compute_instance.go

```diff
@@ -1664,7 +1664,7 @@ func resourceComputeInstanceUpdate(d *schema.ResourceData, meta interface{}) err
 		desiredStatus := d.Get("desired_status").(string)
 
 		if statusBeforeUpdate == "RUNNING" && desiredStatus != "TERMINATED" && !d.Get("allow_stopping_for_update").(bool) {
-			return fmt.Errorf("Changing the machine_type, min_cpu_platform, service_account, enable_display, shielded_instance_config, " +
+			return fmt.Errorf("Changing the machine_type, min_cpu_platform, service_account, enable_display, shielded_instance_config, scheduling.0.node_affinities " +
```

Comment:
Should this be `scheduling.node_affinities` if scheduling is a nested object?

I'm imagining a user seeing this and being confused about the index in this context

Reply:
Fair enough, changed